cmd/crank/beta/trace/trace.go

```diff
@@ -161,3 +170,15 @@ func (c *Cmd) Run(k *kong.Context, logger logging.Logger) error { //nolint:gocyc
 
 	return nil
 }
+
+func (c *Cmd) getResourceAndName() (string, string, error) {
+	// Split resource kind and name
+	if c.Name != "" {
+		return c.Resource, c.Name, nil
```

Comment:
nit: Should we error out if `Name` is still provided while `Resource` has a `/` inside ? 

This is what I am seeing with kubectl in that case:

```
> kubectl get deploymentruntimeconfig.pkg.crossplane.io/default default
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
```

Reply:
Yeah, thought about it, we are going to error out on the get for the resource, but being explicit is always better, I will add a check.

Reply:
done: https://github.com/crossplane/crossplane/pull/4910/commits/e0622f4d4d213291f4d28775dbc3c4c985b236d6